src/main/java/com/example/restapi/form/advice/ValidateAdviceForm.java

```diff
@@ -0,0 +1,16 @@
+package com.example.restapi.form.advice;
+
+import lombok.Data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+public class ValidateAdviceForm {
+
+    private String type;
+
+    private String message;
+
+    private Map<String,String> invalid_params = new HashMap<>();
```

Comment:
Javaの命名規則に従うようにしてほしいです！
キャメルケースですね👍
```suggestion
    private Map<String,String> invalidParams;
```

Reply:
ありがとうございます。
こちら修正いたしました。
e7a863df0b220cfefc870ad9a767a1c9f4cceed7